queries.sql

```diff
@@ -31,3 +31,26 @@ WHERE NOT name = 'Gabumon';
 SELECT * 
 from animals 
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```

Comment:
- [ ] As per project requirements, you have missed part of the requirements. I am pasting all what you are missing below. Kindly add this part.
- I see some of the transactions in your screenshots, but I cannot find the commands in the queries.sql file
```
**Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
Inside a transaction:
Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
Verify that changes were made.
Commit the transaction.
Verify that changes persist after commit.
Take a screenshot of the results of your actions.
Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual ;)
Take a screenshot of the results of your actions.
Inside a transaction:
Delete all animals born after Jan 1st, 2022.
Create a savepoint for the transaction.
Update all animals' weight to be their weight multiplied by -1.
Rollback to the savepoint
Update all animals' weights that are negative to be their weight multiplied by -1.
Commit transaction
Take a screenshot of the results of your actions.
Remember to add all these transactions to your queries.sql file.**
```